src/main/java/core/basesyntax/service/date/DateConverterImpl.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.date;
+
+import core.basesyntax.service.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DateConverterImpl implements DateConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> rawData) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        if (rawData == null || rawData.isEmpty()) {
+            return transactions;
+        }
+
+        for (String line: rawData) {
+
+            if (line.startsWith("type,fruit,quantity")) {
```

Comment:
Hardcoding the header of the CSV file in the code is not a good practice. It would be better to make it a constant.